packages/kbn-docs-utils/src/api_docs/tests/api_doc_suite.test.ts

```diff
@@ -109,31 +109,35 @@ beforeAll(() => {
   pluginA.manifest.serviceFolders = ['foo'];
   const plugins: PluginOrPackage[] = [pluginA, pluginB];
 
-  const { pluginApiMap, missingApiItems, referencedDeprecations } = getPluginApiMap(
-    project,
-    plugins,
-    log,
-    { collectReferences: false }
-  );
+  const { pluginApiMap, missingApiItems, referencedDeprecations, adoptionTrackedAPIs } =
+    getPluginApiMap(project, plugins, log, { collectReferences: false });
 
   doc = pluginApiMap.pluginA;
 
-  pluginAStats = collectApiStatsForPlugin(doc, missingApiItems, referencedDeprecations);
+  pluginAStats = collectApiStatsForPlugin(
+    doc,
+    missingApiItems,
+    referencedDeprecations,
+    adoptionTrackedAPIs
+  );
   pluginBStats = collectApiStatsForPlugin(
     pluginApiMap.pluginB,
     missingApiItems,
-    referencedDeprecations
+    referencedDeprecations,
+    adoptionTrackedAPIs
   );
 
   mdxOutputFolder = Path.resolve(__dirname, 'snapshots');
-  writePluginDocs(mdxOutputFolder, { doc, plugin: pluginA, pluginStats: pluginAStats, log });
-  writePluginDocs(mdxOutputFolder, {
-    doc: pluginApiMap.pluginB,
-    plugin: pluginB,
-    pluginStats: pluginBStats,
-    log,
-  });
-});
+  await Promise.all([
+    writePluginDocs(mdxOutputFolder, { doc, plugin: pluginA, pluginStats: pluginAStats, log }),
+    writePluginDocs(mdxOutputFolder, {
+      doc: pluginApiMap.pluginB,
+      plugin: pluginB,
+      pluginStats: pluginBStats,
+      log,
+    }),
+  ]);
+}, 15000); // For some reason in local it takes a couple of ms, but in CI it times out :shrug:
```

Comment:
These should definitely be integration tests rather than unit tests.

Reply:
++

I'll add the `jest.integration.js` file to the package and move this file to the `integration_tests` dir.